README.md

```diff
@@ -0,0 +1,112 @@
+# moab-versioning
+
+[![Build Status](https://travis-ci.org/sul-dlss/moab-versioning.svg?branch=master)](https://travis-ci.org/sul-dlss/moab-versioning)
+[![Dependency Status](https://gemnasium.com/badges/github.com/sul-dlss/moab-versioning.svg)](https://gemnasium.com/github.com/sul-dlss/moab-versioning)
+[![Gem Version](https://badge.fury.io/rb/moab-versioning.svg)](https://badge.fury.io/rb/moab-versioning)
+
+
+## Usage
+
+```
+  require 'moab'
+```
+
+#### Configuration
+
+```ruby
+Moab::Config.configure do
+  storage_roots 'yer_first_storage_root_dir_except_last_part_of_path'
+  storage_trunk 'the_last_piece_of_the_path_containing_objects'
+  deposit_trunk 'presumably_last_piece_of_path_where_you_want_to_put_new_objects'
+end
+```
+
+You can alternatively use this syntax:
+
+```ruby
+Moab::Config.storage_trunk = 'my directory'
+```
+
+#### Get Latest Version number
+
+```ruby
+current_version = StorageServices.current_version('666') # where 666 is the id
+expect(current_version).to be_an_instance_of Integer
+```
+
+### Stanford-Specific
+
+#### Configuration
+
+```ruby
+Moab::Config.configure do
+  path_method :druid # valid values are :druid or :druid_tree
+end
+```
+
+- when path_method is `:druid_tree`, expect this directory structure: 'jq/937/jp/0017/jq937jp0017'
+- when path_method is `:druid`, expect this directory structure: 'jq937jp0017'
+
+#### Get Latest Version number
+
+```ruby
+current_version = Stanford::StorageServices.current_version('oo000oo0000') # where oo000oo0000 is the druid
+expect(current_version).to be_an_instance_of Integer
+```
```

Comment:
seems like this is a repeat of the above "Get Latest Version number" section?

Reply:
except it says `Stanford::StorageServices` -- heh.   Which can matter for the druid tree thingy

Reply:
ah, yes, i missed that, sorry!